jdaviz/app.py

```diff
@@ -367,8 +367,9 @@ def _link_new_data(self, reference_data=None, data_to_be_linked=None):
         linked_data = dc[data_to_be_linked] if data_to_be_linked else dc[-1]
 
         try:
+            if linked_data.meta.get("Plugin", None) == 'GaussianSmooth':
+                raise AttributeError
```

Comment:
🤯

Reply:
I thought @kecnry and I agreed to get this in in this PR...😅